pkg/machines/helpers.es6

```diff
@@ -313,16 +313,18 @@ export function mouseClick(fun) {
  * @param delay of timeout in ms
  * @param afterTimeoutHandler called only if promise succeeded after timeout
  * @param afterTimeoutFailHandler called only if promise failed after timeout
+ * @param timeoutPlainHandler called dirrectly after timeout expired
```

Comment:
"dirrectly" typo; for consistency, how about naming that `afterTimeoutHandler`? The "plain" seems odd here - at most this should reflect what promises do and be called "always"?

Reply:
Renamed and merged with the afterTimeouthandler which now is triggered either if promise succeeded or if timeout expired before promise returned. These two previous defined handers should do the same anyway, just close the dialog.